### ISSUE TITLE
Do not rename files

### ISSUE THREAD

**Issue by sdaigle2** [sdaigle2/pdi-database#4]
If we are using github to manage version history, don't rename the files.  In the past you have named the files: PDI Operations Database_fe <date>.accdb.  That is only necessary if we are not using a version manager like this github.  

From now on, there should only be 3 files in github:
1. Readme.md
2. PDI Operations Database_fe.accdb
3. PDI Operations Database_be-DEV.accdb

**Comment by smsu07** [comment]
agreed, I was little confused about the push. from now will follow.

**Comment by smsu07** [comment]
I will clean the files as well.

**Comment by sdaigle2** [comment]
OK, I am making some style changes to the files.  I will commit and push when I complete and send you a message.
